tools/hive/kerberos/templates/yarn-site.xml.template

```diff
@@ -0,0 +1,85 @@
+<configuration>
+    <property>
```

Comment:
Similar comments as for the non-kerberized version. Switching to mustache should allow us to to unify the templates a little.